### PR TITLE
Create Layer Logger for PL

### PR DESCRIPTION
**Pull request by mecevit**
LayerLogger implementation for Pytorch Lightning

### CODE REVIEW

File: layer/logged_data/loggers.py [hunk 1]
```diff
@@ -0,0 +1,357 @@
+from argparse import Namespace
+from pathlib import Path
+from typing import Any, Dict, Generator, List, Mapping, MutableMapping, Optional, Union
+
+import torch
+
+import layer
+
+
+class LayerLogger("pytorch_lightning.loggers.logger.Logger"):
```
Comment (volkangurel) [inline layer/logged_data/loggers.py:10]: Can we call this class `PytorchLightningLayerLogger`?

File: layer/logged_data/loggers.py [hunk 2]
```diff
@@ -0,0 +1,357 @@
+from argparse import Namespace
```
Comment (volkangurel) [inline layer/logged_data/loggers.py:1]: Can we make this a file named `pytorch_lightning.py` under the `loggers/` directory?

File: layer/logged_data/loggers.py (hunk 1 above)
Comment (mecevit) [reply layer/logged_data/loggers.py]: Done

File: layer/logged_data/loggers.py (hunk 2 above)
Comment (mecevit) [reply layer/logged_data/loggers.py]: @cihati won't like it but done 😈

**Review by volkangurel — APPROVED** [review]

**Pull request merged**